Tasks/ANT/anttask.ts

```diff
@@ -78,14 +80,28 @@ function processAntOutputLine(line) {
 
 async function doWork() {
 
-    function enableCodeCoverage() {
+    function execEnableCodeCoverage(): Q.Promise<string> {
+        return enableCodeCoverage()
+            .then(function (resp) {
```

Comment:
enableCodeCoverage returns a promise.  So this should be await.  Since you're warning on failure, you can wrap await call in try catch.  Also not sure why you need a separate function for this.  Why not just handle the error in the enableCodeCoverage method and print the warning?